src/lib/reducer.js

```diff
@@ -11,6 +11,28 @@ function isEqual(left, right) {
   return left.type === right.type && left.value === right.value
 }
 
+function isValueType(type) {
```

Comment:
Explicitness. Previous check was to check whether the type *wasn't* a MathExpression. Since a new type was introduced (CssVariable), that logic didn't hold up anymore. I figured I'd not use any implicit logic in order to safeguard against such collisions in the future.